### ISSUE TITLE
sbt-dependency-graph conflict

### ISSUE THREAD

**Issue by francisdb** [akka/alpakka#2145]
Does this project really have to depend on sbt-dependency-graph, that plugin is more something for your global settings.

with a global `v0.10.0-RC1` I get

```
[info] Loading settings for project alpakka from build.sbt ...
[error] java.lang.NoClassDefFoundError: net/virtualvoid/sbt/graph/ModuleTree
```

I found #1388 but it is not clear to me why this project depends on this specific version.

**Comment by raboof** [comment]
I think it's pulled in by sbt-paradox-apidoc (https://github.com/lightbend/sbt-paradox-dependencies/blob/2bd8a8f187b1172b965faa041c709d252acbdadf/build.sbt#L17), but at least we could update it to a version that's released to maven central there, now? would you be interested in PR'ing?

**Comment by francisdb** [comment]
let me have a look

**Comment by francisdb** [comment]
https://github.com/lightbend/sbt-paradox-dependencies/pull/19

**Comment by ennru** [comment]
Fixed with #2147. Thank you @francisdb!
